src/test/java/org/hydev/mcpm/utils/ColorLoggerTest.java

```diff
@@ -17,6 +18,8 @@ void toStdOut()
     {
         var log = ColorLogger.toStdOut();
         log.accept("&aGreen! &cRed! &bBlue! &rDefault!");
+        printc("&aPrintc!");
+        printfc("&aPrintfc! %.2f", 0.031f);
```

Comment:
Suspiciously, this doesn't look like it's testing anything.

Reply:
yea just covered two more methods... because these can't really be tested